docs/notes_for_developers.rst

```diff
@@ -1,12 +1,10 @@
-=====================
-Developing `hera_sim`
-=====================
+=======================
+Developing ``hera_sim``
+=======================
 
-`hera_sim` broadly follows the best-practices laid out in XXX.
+``hera_sim`` broadly follows the best-practices laid out in XXX.
 
 .. todo:: where is that best-practices doc?
 
 
 All docstrings should be written in `Google docstring format <https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html>`_.
```

Comment:
Not the place to address it, but just want to note that I thought we switched to numpy docstring format (and so this should be changed in the future).

Reply:
Yup, definitely. In the PR that we do that, we should also change this.